js/components/portal-dashboard/feedback/activity-level-feedback-student-rows.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Map } from "immutable";
+import { getFormattedStudentName } from "../../../util/student-utils";
+import AwaitingFeedbackActivityBadgeIcon from "../../../../img/svg-icons/awaiting-feedback-activity-badge-icon.svg";
+import GivenFeedbackActivityBadgeIcon from "../../../../img/svg-icons/given-feedback-activity-badge-icon.svg";
+import UpdateFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/update-feedback-question-badge-icon.svg";
+
+import css from "../../../../css/portal-dashboard/feedback/feedback-rows.less";
+
+interface IProps {
+  activityId: string | null;
+  activityIndex: number;
+  feedbacks: Map<any, any>;
+  feedbacksNeedingReview: Map<any, any>;
+  isAnonymous: boolean;
+  updateActivityFeedback: (activityId: string, activityIndex: number, platformStudentId: string, feedback: any) => void;
+}
+
+export const ActivityLevelFeedbackStudentRows: React.FC<IProps> = (props) => {
+  const { activityId, activityIndex, feedbacks, isAnonymous, updateActivityFeedback} = props;
+
+  const handleFeedbackChange = (studentId: string) => (event: React.FormEvent<HTMLTextAreaElement>) => {
+    if (activityId && studentId != null) {
+      const target = event.currentTarget as HTMLTextAreaElement;
+      updateActivityFeedback(activityId, activityIndex, studentId, {feedback: target.value});
+    }
+  };
+
+  const feedbackRows = feedbacks.map ((feedbackData: Map<any, any>, index: number) => {
+
+    const student = feedbackData.get("student");
+    const studentId = student.get("id");
+    const formattedName = getFormattedStudentName(isAnonymous, student);
+    const activityStarted = feedbackData.get("activityStarted");
+    const hasBeenReviewed = feedbackData.get("hasBeenReviewed");
+    const feedback = feedbackData.get("feedback");
+
+    const awaitingFeedbackIcon = <AwaitingFeedbackActivityBadgeIcon />;
+    const givenFeedbackIcon = <GivenFeedbackActivityBadgeIcon />;
+    const updateFeedbackIcon = <UpdateFeedbackQuestionBadgeIcon />;
```

Comment:
`updateFeedbackIcon ` doesn't appear to be used.  If this is needed for future work, then we should comment it out for now to eliminate the warning.  Otherwise delete if it is no longer needed.